devtools-ci-index.yaml

```diff
@@ -99,10 +99,22 @@
     name: devtools
     jobs:
         - '{ci_project}-{git_repo}':
+            description: |
+                This job validates the syntax and yaml file used by JJB from PullReq's aginst the almighty-jobs repo
+            git_username: almighty
+            git_repo: almighty-jobs
+            ci_project: 'devtools'
+            builders:
```

Comment:
you can't change/add steps to a job in jenkins builders.

let me see if i can find the way I handle this in past.